src/utils/parseQueryString.ts

```diff
@@ -0,0 +1,4 @@
+import queryString from "query-string";
+
+export const parseQueryString = (search: string) =>
```

Comment:
thank you 👍

Reply:
right??! i plan on making a followup pr at some point to replace all other instances of querystring.parse with this util